### ISSUE TITLE
Comparison with other implementations

### ISSUE THREAD

**Issue by zhangmozhe** [kwea123/nerf_pl#53]
Hi, I found your implementation really easy to read compared with the reference repo. Great job. I am considering using this code as our codebase for research. I wonder whether you rigorously compare with the official code in terms of accuracy and performance? Do you make any approximations? 

Thanks!

**Comment by kwea123** [comment]
In terms of code behaviour it is the same. The difference, if there's any, would be model weight initialization (tf vs pytorch) and hyperparameters. The pose normalization is also a bit different because I don't understand what the original repo does...
https://github.com/bmild/nerf/blob/20a91e764a28816ee2234fcadb73bd59a613a44c/load_llff.py#L184-L200

In terms of performance, I only compared the lego example, but the paper trains and evaluates on full resolution which is quite resource-consuming; instead I trained on half of the size. The result is quite similar (taking the image size deficit into account), you can see README, and the model is provided in release.

**Comment by zhangmozhe** [comment]
Thank you very much!
